oauthlib/oauth2/rfc6749/grant_types/authorization_code.py

```diff
@@ -95,10 +95,35 @@ class AuthorizationCodeGrant(GrantTypeBase):
     .. _`Authorization Code Grant`: http://tools.ietf.org/html/rfc6749#section-4.1
     """
 
+    default_response_mode = 'query'
+
     def __init__(self, request_validator=None, refresh_token=True):
         self.request_validator = request_validator or RequestValidator()
         self.refresh_token = refresh_token
 
+        self._authorization_validators = []
+        self._token_validators = []
+        self._code_modifiers = []
+        self._token_modifiers = []
+        self.response_types = ['code']
+
+    def register_response_type(self, response_type):
+        self.response_types.append(response_type)
+
+    def register_authorization_validator(self, validator):
+        self._authorization_validators.append(validator)
+
+    def register_token_validator(self, validator):
+        self._token_validators.append(validator)
+
+    def register_code_modifier(self, modifier):
+        self._code_modifiers.append(modifier)
+
+    def register_token_modifier(self, modifier):
+        self._token_modifiers.append(modifier)
+
+    # END-NEW-FOR-OPENID
```

Comment:
Should this have a `#Start of OpenIDConnect` section? Also, maybe rephrase this because it's new now, but it won't always be new.